charts/hedgedoc/Chart.yaml

```
@@ -23,4 +23,4 @@ annotations:
       image: quay.io/hedgedoc/hedgedoc:1.9.8
   artifacthub.io/changes: |
     - kind: changed
-      description: "Update to HedgeDoc 1.9.8, see [GitHub](https://github.com/hedgedoc/hedgedoc/releases/tag/1.9.8) for the release notes."
+      description: "Update to HedgeDoc 1.9.9, see [GitHub](https://github.com/hedgedoc/hedgedoc/releases/tag/1.9.9) for the release notes."
```

Adding a link to the security advisory is probably a good idea for a security bugfix release:

```suggestion
      description: "Update to HedgeDoc 1.9.9, see [GitHub](https://github.com/hedgedoc/hedgedoc/releases/tag/1.9.9) for the release notes."
      links:
        - name: CVE-2023-38487 / GHSA-7494-7hcf-vxpg
          url: https://github.com/hedgedoc/hedgedoc/security/advisories/GHSA-7494-7hcf-vxpg
```